.gitignore

```diff
@@ -5,7 +5,6 @@
 __pycache__
 cache
 *.egg-info
-.venv
```

Comment:
remove .venv in favour of: https://github.com/OpenAdaptAI/OpenAdapt/pull/289/files#r1252004135

Reply:
See https://github.com/OpenAdaptAI/OpenAdapt/pull/289/files#r1252150636

Reply:
RE-addressed in: [b103e18](https://github.com/OpenAdaptAI/OpenAdapt/pull/344/commits/b103e185667d42c426411f4f79f8c426d16f4d33)

Add `.venv` back.